### PR TITLE
New Request Files cmds

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -880,6 +882,41 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -RequestFilesLinkEnabled
+
+Enables or disables Request Files link on the site.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:887]:
```suggestion
Enables or disables the Request Files link on the site.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 2]
```diff
@@ -880,6 +882,41 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RequestFilesLinkExpirationInDays
+
+Specifies the number of days before an Request files link expires for for the site.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:904]:
```suggestion
Specifies the number of days before a Request Files link expires for the site.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 3]
```diff
@@ -880,6 +882,41 @@ Accept pipeline input: False
...
+
+Value can be from 0 to 730 days.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:906]:
```suggestion
The value can be from 0 to 730 days.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**